base/gen_initializer.bal

```diff
@@ -25,16 +25,34 @@ const FHIR_IG = "international";
 # + return - returns error if error occurred
 function init() returns FHIRError? {
     // Anything to initialize should happen here here
+    // We should create terminology as json string variables
```

Comment:
```suggestion
    // We should create terminology as json string variables.
```

Reply:
Some comments contains fullstops at the end and some doesn't.
It is better if we have one approach